README.md

```diff
@@ -1,2 +1,38 @@
-A sample command-line application with an entrypoint in `bin/`, library code
-in `lib/`, and example unit test in `test/`.
+# [Evoleen](https://www.evoleen-technology.com) Fhir Rest Client
+
+This package helps developers to retrieve and manipulate FHIR data with REST client.
+
+This package does not manage authentication into a fhir server, for this feature exists [Azure Identity](https://pub.dev/packages/azure_identity)
+
+## Example
+
+```
+final fhirRestclient = FhirRestClient(
```

Comment:
to uplevel the package with more pubspec point, would be great to have this as example/example.dart (please look up the exact location, I don't remember). But putting an example file into the expected place helps improve the pubspec score

Reply:
[673cb3](https://github.com/evoleen/fhir_rest_client/pull/1/commits/7673cb3bf786943d3ef7387beb0a801032c4187d)